Psi0: 0.0
V0: 0.0
X0: -8584995.019485276
Y0: -1957256.3461890717
lat0: 0.0
lon0: 0.0
yaw0: 0.0
